src/test/util/data_util/data_util_tests.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
+
+import logging
+import unittest
+import pandas as pd
+
+from typing import Tuple, Callable
+from src.main.util.consts import CODE_TRACKER_COLUMN
+from src.main.util.data_util import crop_data_by_timestamp
+from src.main.util.consts import LOGGER_TEST_FILE, LOGGER_FORMAT
+
+
+END_INDEX = 100
+COLUMN = CODE_TRACKER_COLUMN.TIMESTAMP
+
+
+def get_source_data() -> pd.DataFrame:
+    timestamps = range(0, END_INDEX)
+    return pd.DataFrame({
+            COLUMN.value: timestamps
+        })
+
+
+def get_data_with_deleted_by_start_and_end() -> Tuple[pd.DataFrame, int, int]:
+    start = 5
+    end = 10
+    timestamps = range(start, end + 1)
+    return pd.DataFrame({
+        COLUMN.value: timestamps
+    }), start, end
+
+
+def get_data_with_deleted_by_start() -> Tuple[pd.DataFrame, int, None]:
+    start = 5
+    timestamps = range(start, END_INDEX)
+    return pd.DataFrame({
+        COLUMN.value: timestamps
+    }), start, None
+
+
+def are_equal(df_1: pd.DataFrame, df_2: pd.DataFrame):
```

Comment:
Please, add return type :) Maybe I should worry less about typing, but from my point of view it makes our code looks beautiful 🌺